### PR TITLE
Feature/Add residence permit option

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
Not having it on JS is a major parity issue as Residence Permit option has been available on mobile SDK for a while now.

# Solution
Add support and option for selecting Residence Permit as the Document Type being submitted in the IDV flow.
Screenshot 2020-08-26 at 17 44 29

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [x] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [x] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [n/a] Have any new manual tests been written down or the existing ones changed?
- [x] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1162-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/DocumentSelector/index.js
Comment (eraxyz) [inline src/components/DocumentSelector/index.js:115]: Can't tell what this was for but guessing it wasn't needed anymore?

**Comment by eraxyz** [comment]
Should the documentTypes part of the README also be updated?

File: src/supported-documents/index.js
Comment (kopijunkie) [reply src/supported-documents/index.js:4]: Yeap. Totally a typo. 🙈

File: src/components/DocumentSelector/index.js
Comment (kopijunkie) [reply src/components/DocumentSelector/index.js:115]: Same. I couldn't work out why it was there but at the same time the component it's being passed into is not using it or passing it onward in any way so I removed it.

**Review by ethan605 — APPROVED** [review]
LGTM!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Review by ethan605 — APPROVED** [review]
Re-approve after merging from development.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1162-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1162-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**
